concent_api/common/decorators.py

```diff
@@ -169,9 +175,13 @@ def wrapper(request, client_message, client_public_key, *args, **kwargs):
             try:
                 if database_name is not None:
                     sid = transaction.savepoint(using=database_name)
-                response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
+                try:
+                    response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
+                except DatabaseError as e:
+                    response_from_view = get_response_from_view_with_db_retry(view, e, request, client_message, client_public_key, *args, **kwargs)
```

Comment:
Please split into multiple lines, this one is too long.

Reply:
done